cmd/yesiscan/main.go

```diff
@@ -95,301 +113,398 @@ func CLI(program, version string, debug bool, logf func(format string, v ...inte
 			logf("Hello from purpleidea! This is %s, version: %s", program, version)
 			defer logf("Done!")
 
-			ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
-			defer stop()
-
-			bigIntStr := "" // for our int
-			var quiet bool
-			var regexpPath string
-			// config-path makes no sense here
-			var outputType string
-			var outputPath string
-			var outputS3Bucket string
-			region := s3.DefaultRegion
-			profiles := []string{}
-			backends := make(map[string]bool)
-
-			// load from main config file or xdg if config is empty
-			config, err := GetConfig(c.String("config-path"))
-			if err != nil {
-				return err
-			}
-			if config != nil {
-				if config.Quiet != nil {
-					quiet = *config.Quiet
-				}
-				if config.RegexpPath != nil {
-					regexpPath = *config.RegexpPath
-				}
-				// config-path makes no sense here
-				if config.OutputType != nil {
-					outputType = *config.OutputType
-				}
-				if config.OutputPath != nil {
-					outputPath = *config.OutputPath
-				}
-				if config.OutputS3Bucket != nil {
-					outputS3Bucket = *config.OutputS3Bucket
-				}
-				if config.Region != nil {
-					region = *config.Region
-				}
-				if config.Profiles != nil {
-					profiles = []string{} // erase any previous
-					for _, x := range *config.Profiles {
-						profiles = append(profiles, x)
-					}
-				}
-				if config.Backends != nil {
-					for k, v := range config.Backends {
-						backends[k] = v // copy
-					}
-				}
-			}
+			return App(c, program, version, debug, logf)
+		},
+		Flags:                flags,
+		EnableBashCompletion: true,
 
-			// Command line options override anything in the config.
-			if c.IsSet("quiet") {
-				quiet = c.Bool("quiet")
-			}
-			if c.IsSet("regexp-path") {
-				regexpPath = c.String("regexp-path")
-			}
-			// config-path makes no sense here
-			if c.IsSet("output-type") {
-				outputType = c.String("output-type")
-			}
-			if c.IsSet("output-path") {
-				outputPath = c.String("output-path")
-			}
-			if c.IsSet("output-s3bucket") {
-				outputS3Bucket = c.String("output-s3bucket")
-			}
-			if c.IsSet("region") {
-				region = c.String("region")
-			}
-			if c.IsSet("profile") {
-				profiles = []string{} // erase any previous
-				for _, x := range c.StringSlice("profile") {
-					profiles = append(profiles, x)
-				}
-			}
+		Commands: []*cli.Command{
+			{
+				Name:    "web",
+				Aliases: []string{"web"},
+				Usage:   "launch a web server mode",
+				Action: func(c *cli.Context) error {
+					logf("Hello from purpleidea! This is %s, version: %s", program, version)
```

Comment:
I see that this is only here because it's been moved from elsewhere, but as the project grows to be owned by the team (and the community!) it will become important to remove personal bits that imply a single owner / BDFL.

Reply:
Moral rights are different in Canada, and this kind of hello message is not at all unexpected in many projects. Run `vim` for example.

Reply:
Both of those comments say the same thing. This project is from an individual/bdfl.

Reply:
> Moral rights are different in Canada

As soon as you start accepting contributions, moral rights includes the contributor's rights. So it becomes "Hello from the yesiscan contributors" (which includes you!).

Reply:
> As soon as you start accepting contributions, moral rights includes the contributor's rights. So it becomes "Hello from the yesiscan contributors" (which includes you!).

Yup. The top of each source file explicitly says so.

A random hello message from the original author is not unusual in open source. Think of it like an easter egg. Bill gates even put his photo and name in some early software that he is definitely not the main author of. But being a creator of a new project gives you some special privileges. If you'd like to discuss this further, the correct place is in a new issue, not here where this is a whitespace change. As a result I am resolving this thread.